### PR TITLE
PAT-292 Change styling on cim task list to have green buttons

### PR DESCRIPTION
**Pull request by yasinmustafa**

### CODE REVIEW

File: server/routes/index.js [hunk 1]
```diff
@@ -21,11 +21,9 @@ module.exports = ({ offenderService, pathfinderService }) => router => {
...
   // build route for each section
   cimConfig.sections.forEach(section => {
-    section.tasks.forEach(task => {
-      const path = task.id.replace(/([a-z])([A-Z])/g, '$1-$2').toLowerCase()
-      get(cimPath(path), pathfinderCim.view('cimForm', task))
-      post(cimPath(path), pathfinderCim.submit('cimForm', task))
-    })
+    const path = section.id.replace(/([a-z])([A-Z])/g, '$1-$2').toLowerCase()
+    get(cimPath(path), pathfinderCim.view('cimForm', section))
+    post(cimPath(path), pathfinderCim.submit('cimForm', section))
```
Comment (timharrison-moj) [inline server/routes/index.js]: Has a level been removed? Cim sections have tasks, but no longer looping around them?

File: server/views/pages/cimList.html [hunk 2]
```diff
@@ -14,57 +14,52 @@
...
-                                </ul>
-                            </li>
-
+                            {% if section.hideFromCimList !== true %}
+                                {% set path = ["/cimList",bookingId, section.path ] | join('/') %}
+                                {% set taskStarted = data.form[section.id] | length > 0 %}
+                                {% set taskCompleted = data.form[task.id].taskCompleted %}
+                                {% if task.hideFromCimList !== true %}
+                                    {{ cimTask(section.text, taskStarted, path)}}
+                                {% endif %}
```
Comment (timharrison-moj) [inline server/views/pages/cimList.html]: Not sure that I understand where all those styling elements have gone here?! Probably my understanding though.

File: server/routes/index.js (hunk 1 above)
Comment (yasinmustafa) [reply server/routes/index.js]: yes.. previously the there were sections with headings and tasks, each task had a hyperlink... now it's just tasks and each task has a green button

File: server/views/pages/cimList.html (hunk 2 above)
Comment (yasinmustafa) [reply server/views/pages/cimList.html]: the styling here was for the more complex hyperlink style and headings... all that goes away now that we have reverted to simple green buttons

**Pull request merged**